### DISCUSSION TITLE
DiskANN local storage

### DISCUSSION THREAD

**Discussion by rere950303** [milvus-io/milvus#33379] [Q&A and General discussion]
Local storage is used to reduce minio I/O in the diskann index, but if the ephemeral storage limit is reached, a failure occurs, or the performance is simply degraded through minio I/O?? I just wonder what happen if ephemeral storage limit exceed when using diskann index

**Comment by xiaofan-luan** [comment]
diskann store data on local disk, it has to be nvme. 
1. local disk size must enough, and it must be nvme disk.
2. minio disk size must be neough, and io doesn't really matter

**Reply by rere950303** [reply]
@xiaofan-luan
 if ephemeral storage limit exceed when using diskann, the querynode pod evicted???
i can use limited local ephemeral storage capacity. .

**Reply by xiaofan-luan** [reply]
by saying  ephemeral storage do you mean minio disk or milvus used disk?

1. if minio disk full, milvus can not write successfuly, and will abort itself.
2. if local caching is full, milvus pod will be evicted

**Reply by rere950303** [reply]
thank you for replying 🙏
